### PR TITLE
style the LeadeerBord 

### PR DESCRIPTION
**Pull request by majdalkilany**
Add style to the Leaderboard

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -24,9 +24,12 @@ <h2 class="sub-header">Recent scores</h2>
...
       <form class="form-score">
         <h2 class="sub-header">Add your score</h2>
-        <input type="text" placeholder="Your name" class="name-input" /><br><br>
-        <input type="text" placeholder="Score" class="score-input" /><br><br>
-        <button id="submit-score" type="submit">Submit</button>
+        <input type="text" placeholder="Your name" class="name-input input-field" /><br><br>
+        <input type="text" placeholder="Score" class="score-input input-field" /><br><br>
+        <div>
+
+          <button id="submit-score" type="submit">Submit</button>
```
Comment (elisha2kyakpo1) [inline src/index.html:31]: - [ ] Kindly align the submit button in the form with the input fields, kindly note that in this milestone we have to follow the given wireframe design.

leaderboard

submit

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @majdalkilany, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ The PR title and description are good and professional.
✔️ No linter errors.
✔️ Nice UI👏
👏 The app works as expected.

## Required Changes ♻️
_Check the comments under the review._

- [ ] In order to have a more professional readme file, the project title and description should be in line with the current project. Kindly and a meaningful title and description to your readme file.

title


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by majdalkilany** [comment]
**fixed the required changes**

**Review by danmainah — APPROVED** [review]
Hi @leloufadel, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉

**Pull request merged**